mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
+						<h1> Name :{this.props.userData.name} </h1>
```

Comment:
This code adds 'address' to the Profile component, but it's not being passed in as props. Is this intentional? If not, consider removing it to avoid confusion.